build_scripts/windows/azure-cli.wixproj

```diff
@@ -1,14 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!-- Local WiX -->
-  <PropertyGroup>
-    <LocalWixRoot>artifacts\wix\</LocalWixRoot>
-    <WixToolPath>$(LocalWixRoot)</WixToolPath>
-    <WixTargetsPath>$(WixToolPath)Wix.targets</WixTargetsPath>
-    <WixTasksPath>wixtasks.dll</WixTasksPath>
```

Comment:
Shouldn't this be `$(WixToolPath)wixtasks.dll`?

Reply:
No. The wix targets will canonicalize as needed.